Manual.md

```diff
@@ -1302,6 +1326,360 @@ cache during the install/removal of the package
+gdk-pixbuf package it removes the cache file, if present, present at
+`usr/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache`.
+
+It can be added by defining `gdk-pixbuf-loaders` in the `triggers` variable. it is also
+added automatically to any package that has the path `usr/lib/gdk-pixbuf-2.0/2.10.0/loaders`
+available as a directory.
+
+<a id="triggers_gio_modules"></a>
+#### gio-modules
+
+The gio-modules trigger is responsible for updating the Glib GIO module cache with
+`gio-querymodules` from the `glib` package
+
+During install and removal it just runs `gio-querymodules` updating the cache file
+present at `usr/lib/gio/modules`.
+
+It is automatically added to packages that have `/usr/lib/gio/modules` present
+as a directory.
+
+<a id="triggers_gsettings_schemas"></a>
+#### gsettings-schemas
+
+The gsettings-schemas trigger is responsible for compiling Glib's GSettings XML
+schema files during installation and removing the compiled files during removal.
+
+During installation it uses `glib-compile-schemas` from `glib` to compile the
+schemas into files with the suffix .compiled into `/usr/share/glib-2.0/schemas`.
+
+During removal of the glib package it deletes all files inside
+`/usr/share/glib-2.0/schemas` that end with .compiled.
+
+It is automatically added to packages that have `/usr/share/glib-2.0/schemas` present
+as a directory.
+
+<a id="triggers_gtk_icon_cache"></a>
+#### gtk-icon-cache
+
+The gtk-icon-cache trigger is responsible for updating the gtk+ icon cache.
+
+During installation it uses `gtk-update-icon-cache` to update the icon cache.
+
+During removal of the gtk+ package it deletes the `icon-theme.cache` file
+in the directories defined by the variable `gtk_iconcache_dirs`.
+
+It is automatically added on packages that have `/usr/share/icons` available
+as a directory, all directories under that directory have their absolute path
+passed to the trigger.
+
+<a id="triggers_gtk_immodules"></a>
+#### gtk-immodules
+
+The gtk-immodules trigger is responsible for updating the IM (Input Method) modules
+file for gtk+.
+
+During installation it uses `gtk-query-immodules-2.0 --update-cache` to update the
+cache file. It also removes the obsolete configuration file  `etc/gtk-2.0/gtk.immodules`
+if present.
+
+During removal of the `gtk+` package it removes the cache file which is located at
+`usr/lib/gtk-2.0/2.10.0/immodules.cache`.
+
+It is automatically added to packages that have `/usr/lib/gtk-2.0/2.10.0/immodules`
+present as a directory.
+
+<a id="triggers_gtk_pixbuf_loaders"></a>
+#### gtk-pixbuf-loaders
+
+gtk-pixbuf-loaders is the old name for the current `gdk-pixbuf-loaders` trigger and is
+in the process of being removed. It currently re-execs into `gdk-pixbuf-loaders` as a
+compatibility measure.
+
+For information about how it works refer to [gdk-pixbuf-loaders](#triggers_gdk_pixbuf_loaders).
+
+<a id="triggers_gtk3_immodules"></a>
+#### gtk3-immodules
```

Comment:
Maybe it'd make sense to have this right after gtk-immodules.

Reply:
I follow the order that is present when doing

ls /usr/libexec/xbps-triggers